rllib/policy/eager_tf_policy_v2.py

```diff
@@ -155,6 +157,36 @@ def enable_eager_execution_if_necessary():
         if tf1 and not tf1.executing_eagerly():
             tf1.enable_eager_execution()
 
+    @ExperimentalAPI
```

Comment:
Just to clear this up: This method curresponds to the the `maybe_add_time_dimension` method of Policy.
For recurrentl RLModules, it provides a fold and unfold operator.